test.js

```diff
@@ -0,0 +1,50 @@
+var winston = require('winston');
+var assert = require('assert');
+var should = require('should'); 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['should' is defined but never used.](https://www.codacy.com/app/jeffhann/git-mirror-sync/file/750685228/issues/source?bid=2446500&fileBranchId=2446713#l3)